File: web/src/model/victron_controller/dashboard/from_0_2_0_command_set-bookkeeping.ts
// @ts-nocheck
import {SetBookkeeping as v0_2_0_SetBookkeeping} from './v0_2_0/Command'
import {SetBookkeeping as dashboard_SetBookkeeping} from './Command'

export function convert__command__set_bookkeeping__from__0_2_0(from: v0_2_0_SetBookkeeping): dashboard_SetBookkeeping {
    return new dashboard_SetBookkeeping (
        JSON.parse(JSON.stringify(from.key)),
        JSON.parse(JSON.stringify(from.value))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_command_set_bookkeeping.rs


pub fn convert__command__set_bookkeeping__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::command::SetBookkeeping) -> crate::victron_controller::dashboard::command::SetBookkeeping {
    crate::victron_controller::dashboard::command::SetBookkeeping {
        key: serde_json::from_value(serde_json::to_value(&from.key).unwrap()).unwrap(),
        value: serde_json::from_value(serde_json::to_value(&from.value).unwrap()).unwrap(),
    }
}